backend/amp-live-list.go

```diff
@@ -157,13 +163,56 @@ func createMetadata(r *http.Request) []BlogPosting {
 	return result
 }
 
-func createLiveBlogSample(newStatus int, timestamp time.Time, r *http.Request) LiveBlogSample {
+func createLiveBlogSample(newStatus int, timestamp time.Time, r *http.Request, page Page) LiveBlogSample {
+
+	firstBlogID := r.URL.Query().Get("from")
 	if newStatus > len(blogs) {
 		newStatus = len(blogs)
 	}
 	blogItems := getBlogEntries(newStatus, timestamp)
 	score := createScore(newStatus, 0)
-	return LiveBlogSample{BlogItems: blogItems, FootballScore: score, BlogMetadata: createMetadata(r)}
+	firstItemIndex, _ := getProductIndexFromID(firstBlogID, blogItems)
```

Comment:
Maybe I should remove the error, if the item cannot be found at the index provided, it should simply show the first page. What do you think?